server/db/migrationsCreateTables.js

```diff
@@ -17,26 +22,39 @@ console.log('Creating tables...');
         admin BOOLEAN DEFAULT FALSE,
         createdon TIMESTAMPTZ DEFAULT NOW())`);
 
-        await pool.query(`CREATE TABLE IF NOT EXISTS admin(
-        id SERIAL PRIMARY KEY,
-        firstname VARCHAR(50) NOT NULL,
-        lastname VARCHAR(50) NOT NULL,
-        username VARCHAR(50) UNIQUE NOT NULL,
-        email VARCHAR(100) UNIQUE NOT NULL,
-        phonenumber VARCHAR(11) NOT NULL,
-        password VARCHAR(100) NOT NULL,
-        admin BOOLEAN DEFAULT TRUE,
-        createdon TIMESTAMPTZ DEFAULT NOW())`);
+        const password = bcrypt.hashPassword(process.env.ADMIN_PASSWORD);
+        const Admin = {
+            firstname: 'legolas',
+            lastname: 'Aragorn',
+            username: 'ShirePrince',
+            email: 'legolaslegit@gmail.com',
+            password,
+            phonenumber: '08135266484',
+        };
+
+        const queryPlaceholder = `INSERT INTO users (firstname, lastname, username, email, password,  phonenumber)
+       VALUES ($1, $2, $3, $4, $5, $6) RETURNING id, firstname, lastname, username, email, phonenumber`;
+        const values = [Admin.firstname, Admin.lastname, Admin.username,
+            Admin.email, Admin.password, Admin.phonenumber,
+        ];
+        await pool.query(queryPlaceholder, values);
+        // eslint-disable-next-line quotes
+        const makeAdmin = `UPDATE users SET admin = true WHERE email = $1`;
+
+        const adminValue = [Admin.email];
+        await pool.query(makeAdmin, adminValue);
+
 
+        console.log('Creating meetups table...');
         await pool.query(`CREATE TABLE IF NOT EXISTS meetups(
         id SERIAL PRIMARY KEY,
         topic VARCHAR(255) NOT NULL,
         location TEXT NOT NULL,
-        date VARCHAR(50) NOT NULL,
+        happeningon VARCHAR(50) NOT NULL,
```

Comment:
Either use camelCase or snake_case for variable name. Essentially snake_case only for db fields and camelCase everywhere else